src/modules/dashboard/components/TransferFundsDialog/TransferFundsDialogForm.tsx

```diff
@@ -324,6 +317,13 @@ const TokensMoveDialogForm = ({
           )}
         </div>
       </DialogSection>
+      <DialogSection>
+        <Textarea
+          appearance={{ theme: 'fat', resizable: 'vertical' }}
```

Comment:
Please add the `grey` color schma here:
```suggestion
          appearance={{ theme: 'fat', resizable: 'vertical', colorSchema: 'grey' }}
```

Reply:
Also do the same for the other input on the page :+1: